### PR TITLE
Localize content/bn/docs/tutorials/hello-minikube.md

### PR DESCRIPTION
**Pull request by sarkartanmay393**
Signed-off-by: Tanmay Sarkar <84321236+sarkartanmay393@users.noreply.github.com>
Localizes ``content/bn/docs/tutorials/hello-minikube.md`` to Bengali
This is an initial PR as may need some improvements!
Tracking issue: #31677

### CODE REVIEW

**Comment by sarkartanmay393** [comment]
Hey @mitul3737, take a look.

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by sarkartanmay393** [comment]
@mitul3737 Take a look on this once.

File: content/bn/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    minikube dashboard
+    ```
+
+3. Katacoda পরিবেশ (Katacoda Environment): টার্মিনাল প্যানেলের শীর্ষে প্লাস ক্লিক করুন, তারপরে ক্লিক করুন **Select port to view on Host 1** ।
+
+4. Katacoda পরিবেশ (Katacoda Environment): `30000` লিখুন এবং **Display Port** এ ক্লিক করুন।
+
+{{< note >}}
+আপনি যখন  `minikube dashboard` কমান্ড ইস্যু করেন, তখন ড্যাশবোর্ড অ্যাড-অন এবং প্রক্সি সক্রিয় হয় এবং প্রক্সিতে সংযোগ করার জন্য একটি ডিফল্ট ওয়েব ব্রাউজার উইন্ডো খোলে।
+আপনি ড্যাশবোর্ড থেকে কুবারনেটস সংস্থান তৈরি করতে পারেন যেমন স্থাপনা বা পরিষেবা।
```
Comment (mitul3737) [inline content/bn/docs/tutorials/hello-minikube.md:60]:
```suggestion
আপনি ড্যাশবোর্ড থেকে কুবারনেটিস সংস্থান তৈরি করতে পারেন যেমন স্থাপনা বা পরিষেবা।
```

File: content/bn/docs/tutorials/hello-minikube.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+আপনি যদি  `root` এনভায়রনমেন্টে কমান্ড নির্বাহ করছেন, তাহলে [URL ব্যবহার করে ড্যাশবোর্ড অ্যাক্সেস করা] (#open-dashboard-with-url) পড়ুন।
+
+ডিফল্টরূপে, ড্যাশবোর্ড শুধুমাত্র কুবারনেটিস অভ্যন্তরীণ ভার্চুয়াল নেটওয়ার্ক থেকে অ্যাক্সেসযোগ্য।
+
+`dashboard` কমান্ড কুবারনেটিস ভার্চুয়াল নেটওয়ার্কের বাইরে থেকে ড্যাশবোর্ড অ্যাক্সেস করার জন্য একটি অস্থায়ী প্রক্সি তৈরি করে।
+
+আপনি `Ctrl+C` টিপে প্রক্সি থেকে প্রস্থান করতে পারেন।
+কমান্ডটি শেষ হওয়ার পরে, ড্যাশবোর্ডটি কুবারনেটইস ক্লাস্টারে চলতে থাকে।
```
Comment (mitul3737) [inline content/bn/docs/tutorials/hello-minikube.md:69]:
```suggestion
কমান্ডটি শেষ হওয়ার পরে, ড্যাশবোর্ডটি কুবারনেটিস ক্লাস্টারে চলতে থাকে।
```

File: content/bn/docs/tutorials/hello-minikube.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+
+আপনি যদি ওয়েব ব্রাউজারটি স্বয়ংক্রিয়ভাবে খুলতে না চান তবে আপনি `--url` ফ্ল্যাগ দিয়ে নিম্নলিখিত কমান্ডটি কার্যকর করে ড্যাশবোর্ড অ্যাক্সেস URL মুদ্রণ করতে পারেন :
+
+```shell
+minikube dashboard --url
+```
+
+## ডিপ্লয়মেন্ট (Deployment) তৈরি করুন
+
+কুবারনেটিস *পডস*(/en/docs/concepts/workloads/pods/) নেটওয়ার্কিং উদ্দেশ্যে এক বা একাধিক পাত্রের একটি গ্রুপ একসাথে গোষ্ঠীবদ্ধ করে।
```
Comment (mitul3737) [inline content/bn/docs/tutorials/hello-minikube.md:83]:
```suggestion
কুবারনেটিস [পডস](/bn/docs/concepts/workloads/pods/)  নেটওয়ার্কিং উদ্দেশ্যে এক বা একাধিক পাত্রের একটি গ্রুপ একসাথে গোষ্ঠীবদ্ধ করে।
```

File: content/bn/docs/tutorials/hello-minikube.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+এই টিউটোরিয়ালের পডটিতে (pod) শুধুমাত্র একটি পাত্র রয়েছে। কুবারনেটিস
+[*Deployment*](/en/docs/concepts/workloads/controllers/deployment/) হলো পডের
```
Comment (mitul3737) [inline content/bn/docs/tutorials/hello-minikube.md:85]:
```suggestion
[Deployment](/bn/docs/concepts/workloads/controllers/deployment/) হলো পডের
```

File: content/bn/docs/tutorials/hello-minikube.md [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+এটি স্বাস্থ্য পরীক্ষা করে এবং পডের ধারকটি বন্ধ হয়ে গেলে পুনরায় চালু করে। পড তৈরি এবং স্কেলিং পরিচালনা করার উপায় হিসাবে স্থাপনের সুপারিশ করা হয়।
```
Comment (mitul3737) [inline content/bn/docs/tutorials/hello-minikube.md:86]:
```suggestion
একটি স্বাস্থ্য পরীক্ষা করে এবং পডের ধারকটি বন্ধ হয়ে গেলে পুনরায় চালু করে। পড তৈরি এবং স্কেলিং পরিচালনা করার উপায় হিসাবে স্থাপনের সুপারিশ করা হয়।
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]
Looks good to me but need one more change

File: content/bn/docs/tutorials/hello-minikube.md (hunk 1 above)
Comment (sarkartanmay393) [reply content/bn/docs/tutorials/hello-minikube.md:60]: I can't add these suggestions to a batch, is there any issue?

File: content/bn/docs/tutorials/hello-minikube.md (hunk 5 above)
Comment (sarkartanmay393) [reply content/bn/docs/tutorials/hello-minikube.md]: @mitul3737 check it out.

**Review by mitul3737 — APPROVED** [review]
lgtm

**Comment by mitul3737** [comment]
tide/merge-method-squash

**Pull request merged**